README.md

```diff
@@ -1,138 +1,23 @@
 # Microservices March Demo Architecture Platform (WIP)
-
 Shared infrastructure for the Microservices March demo architecture.
 
 ## Usage
+The aim is to set things up as manually as possible without involving something heavy like Kubernetes. Future iterations might have examples of more advanced setups.
 
-The aim is to set things up as manually as possible without creating "magic" via Docker Compose or involving something heavy like Kubernetes.  Future iterations might have examples of more advanced setups.
-
-### Prerequisites
-
-Make sure you have this repo as well as the following additional repos checked out:
-
-* [notifier](https://github.com/microservices-march/notifier)
-* [messenger](https://github.com/microservices-march/messenger)
-
-### Start the Shared Platform Infrastructure
-
-From this repository, run:
-
-```bash
-docker-compose up -d
-```
-
-### Start the `messenger` Service
-
-1. From the `messenger` repository, build the Docker image:
-
-    ```bash
-    docker build -t messenger .
-    ```
-
-2. From the `messenger` repository, start the PostgreSQL database:
-
-    ```bash
-    docker-compose up -d
-    ```
-
-3. Start the `messenger` service in a container:
-
-    ```bash
-    docker run -d -p 8083:8083 --name messenger -e PGPASSWORD=postgres -e CREATE_DB_NAME=messenger -e PGHOST=messenger-db-1 -e AMQPHOST=rabbitmq -e AMQPPORT=5672 -e PORT=8083 --network mm_2023 messenger
-    ```
-
-4. SSH into the container to set up the PostgreSQL DB:
-
-    ```bash
-    docker exec -it messenger /bin/bash
-    ```
-
-5. Create the PostgreSQL DB:
-
-    ```bash
-    PGDATABASE=postgres node bin/create-db.mjs
-    ```
-
-6. Create the PostgreSQL DB tables:
-
-    ```bash
-    node bin/create-schema.mjs
-    ```
-
-7. Create some PostgreSQL DB seed data:
-
-    ```bash
-    node bin/create-seed-data.mjs
-    ```
+## Running just shared infrastructure
+If you are running some subset of the services manually, just run `docker-compose up` from this repository to start up the message queue.  This does not set up the NGINX load balancer.
```

Comment:
```suggestion
```
You will already cover the manual setup in [L15](https://github.com/microservices-march/platform/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15)

Reply:
This one is a bit different but maybe I've worded it poorly.  There's three things you can do:
1. Manual setup (prebuilt containers everywhere, started with mostly `docker run` commands and some `docker-compose up` commands).
2. Quick setup (one big docker compose file)
3. Running a subset of services locally on your machine (how most devs work in my experience).  So for example I might be running `messenger` with just node on my mac, and need to have the rabbit queue available for that.  

How does this alternative wording sound?

> If you are working on one of the services, such as [messenger](https://github.com/microservices-march/messenger), that depends on a piece of shared infrastructure to function, you can run `docker-compose up` from this repository to provide that infrastructure.  This does not set up the NGINX load balancer.

Reply:
Your suggestion should work! The original wording was indeed unclear/missing some info.